src/descriptor/policy.rs

```diff
@@ -822,6 +834,17 @@ impl<Ctx: ScriptContext> ExtractPolicy for Miniscript<DescriptorPublicKey, Ctx>
                         csv: Some(*value),
                     },
                 };
+                if let BuildSatisfaction::PsbtTimelocks {
+                    current_height,
+                    input_max_height,
+                    ..
+                } = build_sat
+                {
+                    let older = Older::new(Some(current_height), Some(input_max_height), false);
+                    if Satisfier::<bitcoin::PublicKey>::check_older(&older, *value) {
+                        policy.satisfaction = policy.contribution.clone();
```

Comment:
I think here we should also check that the transaction has the right nSequence, which can be done with the `PsbtInputSatisfier` from `miniscript`: https://docs.rs/miniscript/5.1.0/src/miniscript/psbt/mod.rs.html#268-282

If both are `true`, then the timelock can be used

Reply:
should be fixed in 6558c74d485df0d0bcd64a1ca108c40b5e5a04da